var/spack/repos/builtin/packages/darshan-runtime/package.py

```diff
@@ -115,9 +115,9 @@ def configure_args(self):
         if "+apmpi" in spec:
             extra_args.append("--enable-apmpi-mod")
         if "+apmpi_sync" in spec:
-            extra_args.append(["--enable-apmpi-mod", "--enable-apmpi-coll-sync"])
+            [extra_args.append(x) for x in ["--enable-apmpi-mod", "--enable-apmpi-coll-sync"]]
```

Comment:
Why not just do `extra_args.extend(["--enable-apmpi-mod", "--enable-apmpi-coll-sync"])`? That would be more readable IMO.

Reply:
There are multiple ways to do it, and this was just my preference.

Reply:
Please do act on Michael's advice :) doesn't make sense to write it like this, it's allocating an array [None, None]?

Reply:
He didn't ask me to change it or give advice, he just said "Why not do" and I answered his question. If he wanted me to change it perhaps say that directly next time? Thanks!

Reply:
Can you please change it? Thanks!

Reply:
Already did 30 minutes ago :)

Reply:
Ok great! Thanks!